engine/src/main/java/com/datatorrent/stram/client/StramClientUtils.java

```diff
@@ -118,6 +118,8 @@
   public static final long DELEGATION_TOKEN_MAX_LIFETIME_DEFAULT = 7 * 24 * 60 * 60 * 1000;
   public static final String TOKEN_REFRESH_PRINCIPAL = StramUserLogin.DT_AUTH_PREFIX + "token.refresh.principal";
   public static final String TOKEN_REFRESH_KEYTAB = StramUserLogin.DT_AUTH_PREFIX + "token.refresh.keytab";
+  public static final String CUSTOM_SSL_KEYSTORE_CONFIG = StreamingApplication.APEX_PREFIX + "ssl.keystore.config";
```

Comment:
I would like to keep this PR restricted to that the JIRA expects. So my proposal for this PR is as follows:

Support both dt.ssl.keystore.config and apex.ssl.keystore.config with the understanding that dt.ssl.keystore.config is deprecated and will be removed in the next major release (4.x).

Enhance Javadocs for the new members added in this PR with proper explanation about usage and deprecation.

Regarding naming of the keys I am awaiting a response from @PramodSSImmaneni  to make sure we have an agreement on the naming.